packages/lesswrong/components/review/FrontpageReviewWidget.tsx

```diff
@@ -140,51 +140,57 @@ const FrontpageReviewPhase = ({classes}: {classes: ClassesType}) => {
+
   // EA will use LW text next year, so I've kept the forumType genericization
   const overviewToolip = isEAForum ?
-    <>
+    <div>
       <div>The EA Forum is reflecting on the best EA writing, in three phases</div>
       <ul>
-        <li><em>Nomination</em> ({nominationStartDate.format('MMM Do')} – {nominationEndDate.format('MMM Do')})</li>
-        <li><em>Review</em> ({nominationEndDate.clone().add(1, 'day').format('MMM Do')} – {reviewEndDate.format('MMM Do')})</li>
-        <li><em>Voting</em> ({reviewEndDate.clone().add(1, 'day').format('MMM Do')} – {voteEndDate.format('MMM Do')})</li>
+        <li><em>Nomination</em> ({nominationPhaseDateRange})</li>
+        <li><em>Review</em> ({reviewPhaseDateRange})</li>
+        <li><em>Voting</em> ({votingPhaseDateRange})</li>
       </ul>
       <div>To be eligible, posts must have been posted before January 1st, 2021.</div>
       <br/>
       {/* TODO; this won't be true in other phases */}
       <div>(Currently this section shows a random sample of {REVIEW_YEAR} posts, weighted by karma)</div>
-    </> :
-    <>
-      <div>The ${forumTitleSetting} community is reflecting on the best posts from {REVIEW_YEAR}, in three phases</div>
+    </div> :
+    <div>
+      <div>The {forumTitle} community is reflecting on the best posts from {REVIEW_YEAR}, in three phases:</div>
       <ul>
-        <li><em>Nomination</em> ({nominationStartDate.format('MMM Do')} – {nominationEndDate.format('MMM Do')})</li>
-        <li><em>Review</em> ({nominationEndDate.clone().add(1, 'day').format('MMM Do')} – {reviewEndDate.format('MMM Do')})</li>
-        <li><em>Voting</em> ({reviewEndDate.clone().add(1, 'day').format('MMM Do')} – {voteEndDate.format('MMM Do')})</li>
-        {!isEAForum && <li>The LessWrong moderation team will incorporate that information, along with their judgment, into a "Best of {REVIEW_YEAR}" book.</li>}
+        <li><em>Rough Voting</em> ({nominationPhaseDateRange})</li>
+        <li><em>Review</em> ({reviewPhaseDateRange})</li>
+        <li><em>Final Voting</em> ({votingPhaseDateRange})</li>
       </ul>
+      <div>The {forumTitle} moderation team will incorporate that information, along with their judgment, into a "Best of {REVIEW_YEAR}" book.</div>
       {/* TODO; this won't be true in other phases */}
       <div>(Currently this section shows a random sample of {REVIEW_YEAR} posts, weighted by karma)</div>
-    </>
+    </div>
 
   const nominationsTooltip = isEAForum ?
-    <>
+    <div>
       <div>Nominate posts for the EA Writing Mega Review</div>
       <ul>
         <li>Any post from before 2021 can be nominated</li>
         <li>Any user registered before the start of the review can nominate posts</li>
-        <li>A post requires two nominations to progress to the review phase</li>
+        <li>Posts with at least one vote proceed to the Review Phase.</li>
```

Comment:
```suggestion
        <li>Posts with at least one positive vote proceed to the Review Phase.</li>
```

Reply:
I'm not sure we should bother doing this, although I do see the appeal. By the time we get to the review phase we'll already know which posts were overall sorted low, and we'll already need to encourage people to basically ignore low-ranked posts unless they think they're a hidden gem.

Reply:
It already does it tho, check out the denormalized field filter